src/libs/SidebarUtils.ts

```diff
@@ -77,14 +77,18 @@ function getOrderedReportIDs(
 
     // Filter out all the reports that shouldn't be displayed
     let reportsToDisplay = allReportsDictValues.filter((report) => {
-        const parentReportActionsKey = `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report?.parentReportID}`;
+        if (!report) {
+            return false;
+        }
+
+        const parentReportActionsKey = `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.parentReportID}`;
         const parentReportActions = allReportActions?.[parentReportActionsKey];
         const reportActions = ReportActionsUtils.getAllReportActions(report.reportID);
```

Comment:
@aimane-chnaif  can you consider accessing the report directly from `allReports`? It serves the same purpose.

```ts

const reportActionsKey = `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report?.reportID}`;
const reportActions = allReportActions?.[reportActionsKey];

```